topics/kmm-plugin-releases.md

```diff
@@ -34,12 +34,39 @@ Release info
 Release highlights
 </th>
 <th>
-Compatible Kotlin plugin version
+Compatible Kotlin version
 </th>
 </tr>
 <tr>
 <td>
 
+**0.2.1**
+
+Released: February 15, 2021
+
+</td>
+<td>
+
+* [Support for Kotlin 1.4.30](https://youtrack.jetbrains.com/issue/KT-44038).
+* [New compatibility policy – for each major version of Kotlin there will be a new version of KMM plugin](https://youtrack.jetbrains.com/issue/KT-43211). (TODO TBD remove this item?)
```

Comment:
For Kotlin, we don't use "major" or "minor" version. We have feature, incremental, and bug-fix releases. Read https://kotlinlang.org/docs/releases.html

Reply:
Ok, I wrote so because of translation from Russian "теперь совместимость до следующей мажорной версии". Seems like we use "major" internally despite the convention. Will fix.